features/bevragen/persoon/verblijfplaats/adres/onbekendwaardes.feature

```diff
@@ -10,13 +10,13 @@ Rule: Niet leveren van de standaard/onbekend waarde van een adres veld
   Het gaat om de volgende properties en standaardwaardes van de persoon:
   | property                                         | standaardwaarde  |
   | ------------------------------------------------ | ---------------- |
-  | verblijfplaats.korteNaam                         | .                |
+  | verblijfplaats.korteStraatnaam                   | .                |
```

Comment:
dit verwijst denk ik naar velden in de GbaPersoon? (anders was het verblijfplaats.verblijfadres.korteStraatnaam)
daarin heet dit veld straat, niet korteStraatnaam

Reply:
(was trouwens al fout in vorige versie met verblijfplaats.korteNaam)

is het niet beter in deze tabel de LO namen te gebruiken? dus "straatnaam (11.10)", "huisnummer (11.20)", enz.?

Reply:
Lo namen opgenomen